src/lib/sql/sql_translator.cpp

```diff
@@ -782,6 +794,18 @@ void SQLTranslator::_translate_select_list_groupby_having(const hsql::SelectStat
     if (hsql_expr->type == hsql::kExprStar) {
       AssertInput(_from_clause_result, "Can't SELECT with wildcards since there are no FROM tables specified");
 
+      if (is_aggregate) {
+        // SELECT * is only valid if every input column in named in the GROUP BY clause
```

Comment:
This was handled differently before (including all columns from the group by list), but I believe this is the only correct way to do it. MySQL is weird anyway in that it generally allows non-GROUP BY columns to be added and simply picks the first (or any?) row. Postgres agrees:

http://sqlfiddle.com/#!17/683eb/1

Reply:
Did not know MySQL allows such things. I would have expected the behavior you've apparently implemented (I just read the comment ;) ).